### ISSUE TITLE
Fan control not starting

### ISSUE THREAD

**Issue by RicoW1985** [Rem0o/FanControl.Releases#2842]
Hi, Today i noticed fancontrol is not working at all.
I uninstalled and installed latest version from the website, tried both installers and the zipfile.
it asks permissions but after that nothing happens.
Im running windows 11 with latest updates.
I hope anyone can help, because i really love the program.
thanks in advance
(added screenshot with specs)
![spec](https://github.com/user-attachments/assets/4b0e34fc-9ad4-493d-8045-7ba3c58c8a0a)

**Comment by boogieboard** [comment]
I also have the same issue. I attempted to update via the GUI and nothing comes up anymore.

**Comment by adonaldson** [comment]
In case anyone stumbles onto this in the future, clearing out / backing up my old `userConfig` file allowed the app to start. Then it's a case of re-creating your old settings.

**Comment by RicoW1985** [comment]
adonaldson, unfortunatly this didn't help

**Comment by Rem0o** [comment]
Any crash event in eventviewer? Any logs in logs.txt?

**Comment by N1ckstars** [comment]
yeah i have the same issue too, tried reinstalling it with .net 4.8 and 8, tried using revo uninstaller to remove all the files associated with it, with and without restarts, and still nothing

**Comment by WCDThor** [comment]
Consider figuring out how to revert to V208 or V207. This latest version (V209) has caused me a lot of issues, different from yours though, and when I have time after I'm done with school I'm thinking of reverting if there hasn't been a patch by December 10th.

**Comment by AdiMarianMutu** [comment]
I'm having the same issue, I have an AMD GPU (RX 7900XTX) and I've done a bisect install starting from the last version `209` and found out that the last working version on my system is `194`.

I see this in the `195` version changelogs `New built-in AMD GPU (ADLX) support`. 

I think that the built-in AMD GPU support is clearly failing.

**Comment by Rem0o** [comment]
@AdiMarianMutu 

>  I'd happily try to solve this on my own if you'd make the project open source.

Good news: https://github.com/Rem0o/ADLXWrapper

Otherwise, if there are crash events in your Eventviewer, would be useful.  Also the driver version you are using for your AMD card.

**Comment by RicoW1985** [comment]
im sorry im not that handy with computers to install the adlxwrapper. 

> Application: FanControl.exe
CoreCLR Version: 8.0.1124.51707
.NET Version: 8.0.11
Description: The process was terminated due to an unhandled exception.
Exception Info: exception code c0000005, exception address 00007FF80B4F2BD3

> Naam van toepassing met fout: FanControl.exe, versie: 207.0.0.0, tijdstempel: 0x66e80000
Naam van module met fout: atiadlxx.dll, versie: 32.0.12019.1028, tijdstempel: 0x670985c5
Uitzonderingscode: 0xc0000005
Foutmarge: 0x0000000000062bd3
Id van proces met fout: 0x0x43E0
Starttijd van toepassing met fout: 0x0x1DB44D6D249CDA6
Pad naar toepassing met fout: C:\Program Files (x86)\FanControl\FanControl.exe
Pad naar module met fout: C:\Windows\SYSTEM32\atiadlxx.dll
Rapport-id: 2779fba1-4710-4ed3-aa0d-d07f230fb145
Volledige pakketnaam met fout: 
Relatieve toepassings-id van pakket met fout: 

this is what i get in eventviewer.
running 24.8.1 of amd software.

hope this helps.

thanks in advance

**Comment by Rem0o** [comment]
> running 24.8.1 of amd software

Is there a reason you are not running the latest version? 24.8.1 is busted and causes various crashes.

**Comment by RicoW1985** [comment]
well, fancontrol works again! thanks! 
now I need to figure something else out though. 
![Untitled](https://github.com/user-attachments/assets/2a0ca099-4fba-4493-9bcc-7ffd0199ebda)
the main reason I downgraded to 24.8.1.
I guess i will be gaming on dx11 until it gets fixed

**Comment by AdiMarianMutu** [comment]
> @AdiMarianMutu
> 
> > I'd happily try to solve this on my own if you'd make the project open source.
> 
> Good news, the AMD driver/api portion of it is: https://github.com/Rem0o/ADLXWrapper . There is a test app in there you can play with.
> 
> Otherwise, if there are crash events in your Eventviewer, would be useful. Also the driver version you are using for your AMD card.

I'll take a look once I have some more free time on my hands.

I'm using the `24.8.1` driver as the last one `24.10.1` is pure crap, crashes over crashes (reddit is full of people reporting this).

Here the result of the `Eventviewer` for the `v209`:

```
Faulting application name: FanControl.exe, version: 209.0.0.0, time stamp: 0x67110000
Faulting module name: atiadlxx.dll, version: 32.0.12019.1028, time stamp: 0x670985c5
Exception code: 0xc0000005
Fault offset: 0x0000000000062bd3
Faulting process id: 0x0x44E4
Faulting application start time: 0x0x1DB44787D2369D1
Faulting application path: Z:\Stand Alone Apps\FanControl\FanControl.exe
Faulting module path: C:\Windows\SYSTEM32\atiadlxx.dll
Report Id: 4dade99d-e101-4ed9-a8a7-8b14fe83a4db
Faulting package full name: 
Faulting package-relative application ID: 
```

It seems that the `atiadlxx` dll is the one failing, to be honest I'm not surprised, AMD is the best in providing faulty drivers :)

**Comment by RicoW1985** [comment]
> > @AdiMarianMutu

> I'm using the `24.8.1` driver as the last one `24.10.1` is pure crap, crashes over crashes (reddit is full of people reporting this).

this here is your problem. I had the same issue... now running dx11 for the time being

**Comment by Willxfox** [comment]
Having the same issues. Running v212 with AMD 7900XTX. With ADLX enabled it will not start and just crashes. I roll back to v194 it will start but no ADLX. If you install the ADLX plugin it will not start again. Event log shows the following on v212. AMD driver is 24.12.1

Faulting application name: FanControl.exe, version: 212.0.0.0, time stamp: 0x67110000
Faulting module name: atiadlxx.dll, version: 32.0.12019.1028, time stamp: 0x670985c5
Exception code: 0xc0000005
Fault offset: 0x0000000000062bd3
Faulting process id: 0x1E34
Faulting application start time: 0x1DB725446371B42
Faulting application path: C:\Users\willx\Downloads\FanControl_212_net_8_0\FanControl.exe
Faulting module path: C:\WINDOWS\SYSTEM32\atiadlxx.dll
Report Id: b8fa04d0-cae4-48bd-8756-3b50f48590de

**Comment by Rem0o** [comment]
You may try a full reinstall of the driver, with a DDU clean before + a full "clean" install on top.  Some people have had chance with that.   If you simply upgrade drivers, it seems like some faulty stuff remains from previous versions which leads to crash like this.

**Comment by Willxfox** [comment]
You beat me to it! Just did a clean driver install and this fixed the issues. Thanks!

**Comment by RicoW1985** [comment]
well I think this is solved :)
thanks @Rem0o 
will donate very soon! your work is epic!
